README.md

```diff
@@ -66,11 +112,9 @@ Once you have serialized your sequence, you can send it with the SDK with the fo
 from sdk import SDK
 from pulser import devices, Register, Sequence
 
-username="your_username" # Replace this value by your username or email on the PASQAL platform.
-group_id="your_group_id" # Replace this value by your group_id on the PASQAL platform.
-password="your_password" # Replace this value by your password on the PASQAL platform.
-# Ideally, do not write this password in a script but provide in through the command-line or as a secret environment variable.
-
+username="your_username" 
+group_id="your_group_id" 
```

Comment:
I think we should mention that the user has to go to the portal and fetch it from there

Reply:
Indeed, but that would be repeated with the new Authentication section. But I'll add the comment again for the group id

Reply:
i readded the comments even if i find them redundant with the above authentication comments